### PR TITLE
[VB-182] Replace TravisCI with GitHub actions

### PR DESCRIPTION
**Pull request by ErlendMariusOmmundsen**
Replaces TravisCI Usage with the GitHub action workflow file tests.yml 🤠

### CODE REVIEW

**Comment by ErlendMariusOmmundsen** [comment]
Click [here](https://github.com/vektorprogrammet/vektorprogrammet/actions/runs/4524509881/jobs/7968391863) to see the workflow in action 🤠

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  symfony:
+    name: Symfony 5.4 (PHP ${{ matrix.php-versions }})
```
Comment (Lundli) [inline .github/workflows/tests.yml:12]: Update naming (old repo is not `5.4`)

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    # https://hub.docker.com/_/ubuntu/
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+      matrix:
+        php-versions: ["7.4"]
```
Comment (Lundli) [inline .github/workflows/tests.yml:18]: You should change this to `7.3`. Prod is aaactually running 7.3 (i know....)
```suggestion
        php-versions: ["7.3"]

```

**Review by Lundli — APPROVED** [review]
LGTM

**Pull request merged**